backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -260,29 +241,56 @@ func (fs *experimentStore) GetListView(ctx context.Context) ([]*experimentation.
 
 func (fs *experimentStore) GetExperimentRunDetails(ctx context.Context, id uint64) (*experimentation.ExperimentRunDetails, error) {
 	sqlQuery := `
-        SELECT experiment_run.id, lower(execution_time), upper(execution_time), cancellation_time, creation_time, details FROM experiment_config, experiment_run
+		SELECT 
+			experiment_run.id,
+			lower(execution_time),
+			upper(execution_time),
+			cancellation_time,
+			creation_time,
+			experiment_config.id, 
+			details FROM experiment_config, experiment_run
         WHERE experiment_run.id = $1 AND experiment_run.experiment_config_id = experiment_config.id`
 
 	row := fs.db.QueryRowContext(ctx, sqlQuery, id)
 
-	var fetchedID uint64
-	var startTime, endTime, cancellationTime sql.NullTime
-	var creationTime time.Time
 	var details string
+	run := ExperimentRun{}
+	config := ExperimentConfig{Config: &any.Any{}}
+	err := row.Scan(&run.id, &run.startTime, &run.endTime, &run.cancellationTime, &run.creationTime, &config.id, &details)
```

Comment:
Is there a better way to query DB? Afaik, `scan` affects the performance.

Reply:
frankly speaking I'm not sure - can we leave it the way it's as part of this PR (since it's been like this previously too) and address this if performance is an issue? Reading through the docs for Scan method I do not see any mentions of performance issues (https://golang.org/pkg/database/sql/#Row.Scan and https://golang.org/pkg/database/sql/#Rows.Scan)